test/scripts/sports_data_from_sportradar.txtar

```diff
@@ -0,0 +1,8 @@
+# [execute] attest fn call to SportRadar API
+stdin fn-call.json
+exec bky-as attest-fn-call
+cp stdout result.json
+
+# [check] assert stdout contains correct game information
+exec jq -c -r '.transitive_attested_function_call.claims.output | @base64d | fromjson' result.json
+stdout '{"Success":true,"Error":"","Value":{"points_per_minute":'
```

Comment:
Not sure why, but whenever I provided the full string it would fail to find a match. I compared the output string to the one I provided in different editors and it said they were the same, but test script didn't seem to think so. Thus, I'm just searching for the first part of the sports radar return struct

Reply:
not sure what you're running into. I wonder if there's something going on with newlines or the way that jq is producing the string with the `-c` flag. I suggested an edit below that uses the same pattern for comparing out put files instead.